### ISSUE TITLE
Comparision Filter PropertyIsLike

### ISSUE THREAD

**Issue by rajsync** [geostyler/geostyler-sld-parser#901]
# Feature Request

**Is your feature request related to a problem? Please describe.**
currently wildcard, singleChar and escape values are hard coded.

**Describe the solution you'd like**
option to pass wildcard, singleChar and escape values, while creating comparison filter PropertyIsLike

**Comment by jansule** [comment]
Good point. To do so, we would have to add these options to the typing for the `propertyIsLike` filter in the geostyler-style [here](https://github.com/geostyler/geostyler-style/blob/master/style.ts#L81-L97) and [here](https://github.com/geostyler/geostyler-style/blob/master/style.ts#L61) (the `*=` is the equivalent of `propertyIsLike`). Then, the sld-parser could be adjusted to that.

Are you planning on providing a PR for this?

**Comment by rajsync** [comment]
I have made changes to provide "PropertyIsLike" attributes. Please check the pull request https://github.com/geostyler/geostyler-style/pull/616.

**Comment by rajsync** [comment]
I have to make changes to "geostyler-sld-parser" project to remove use the provided like attributes. After the pull request complete and released?

**Comment by jansule** [comment]
You can also develop based on your unmerged changes by using `npm link`. Please reference the open PR in the description, then.

**Comment by KaiVolland** [comment]
Hey @rajsync do you think you will continue working on the PR in the geostyler-style? Otherwise we might could take care of it in the upcoming codesprint.
